app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,26 @@
+class SessionsController < ApplicationController
+  
+  def new
+    redirect_to home_path if current_user
+  end
+
+  def create
+
+    user = User.where(email: params[:email]).first
```

Comment:
Since Rails 4, `.find_by` is preferred method for searching a single record. `.where(args).first` is less effective.